eb_hooks.py

```diff
@@ -301,7 +301,7 @@ def parse_hook_lammps_remove_deps_for_CI_aarch64(ec, *args, **kwargs):
     """
     Remove x86_64 specific dependencies for the CI to pass on aarch64
     """
-    if ec.name == 'LAMMPS' and ec.version in ('2Aug2023_update2',):
+    if ec.name == 'LAMMPS' and ec.version in ('2Aug2023_update2', '29Aug2024'):
```

Comment:
Since we will still needs this hook for `29Aug2024` it is probably not an issue but for later versions this might become a problem. When the version is not included, the build errors out with:
```
== 2024-09-19 07:39:28,463 build_log.py:171 ERROR EasyBuild crashed with an error (at easybuild/tools/build_log.py:111 in caller_info): Failed to process easyconfig /tmp/eb-3_9md2gk/eb-p410if82/files_pr21436/l/LAMMPS/LAMMPS-29Aug2024-foss-2023b-kokkos.eb: LAMMPS-specific hook triggered for non-LAMMPS easyconfig?! (at easybuild/framework/easyconfig/easyconfig.py:2094 in process_easyconfi
```

Reply:
It's good to be strict here, because we do expect that for a future LAMMPS version this will not be required anymore?